src/app/pages/swap/components/swap-asset-select/swap-asset-select-quote.tsx

```diff
@@ -17,11 +17,14 @@ export function SwapAssetSelectQuote() {
   const [assetField] = useField('swapAssetQuote');
   const navigate = useSwapNavigate();
 
-  const amountAsFiat = useAlexSdkAmountAsFiat(
-    assetField.value?.balance,
-    assetField.value?.price,
-    amountField.value
-  );
+  const amountAsFiat =
+    assetField.value &&
+    amountField.value &&
+    convertInputAmountValueToFiat(
+      assetField.value.balance,
+      assetField.value.price,
+      amountField.value
+    );
```

Comment:
Super ugly, but still better than leaking the possibly undefined logic into `convertInputAmountValueToFiat`

Reply:
...maybe will at least capture in a variable before calling the function. It is ugly.